packages/oui-datagrid/README.md

```diff
@@ -713,6 +717,7 @@ call `rows-loader` and then a `row-loader` call for each line.
 | `selectable-rows`                 | boolean   | <?        | no                  | `true`, `false`  | `false`      | if rows can be selected
 | `columns-parameters`              | array     | <?        | no                  | n/a              | `undefined`  | columns parameters (see below)
 | `on-columns-parameters-change`    | function  | &         | no                  | n/a              | n/a          | triggered on column parameter change when datagrid is customizable
+| `on-row-select`                   | function  | &         | no                  | n/a              | n/a          | triggered when a row is selected
```

Comment:
`&?` if the option is optionnal

Reply:
It's made on purpose. `&` call `angular.noop()` instead of `undefined` (with `&?`). 
It can be very useful inside template, and can avoid an if.

Reply:
Ok nice, it's good to know

Reply:
It's lying to the users though, we could have a `mandatory` column that declares if a parameter is mandatory, independently from the binding